### ISSUE TITLE
Bug in RealIP plugin (REAL_IP_FROM_URLS is ignored)

### ISSUE THREAD

**Issue by gspiliotis** [bunkerity/bunkerweb#419]
It seems there is a typo in `core/realip/jobs/realip-download.py`.  The environment variable `REALIP_FROM_URLS` should be `REAL_IP_FROM_URLS` (there is an underscore missing) in this part of the code:

```
    # Get URLs
    urls = []
    for url in os.getenv("REALIP_FROM_URLS", "").split(" ") :
        if url != "" and url not in urls :
            urls.append(url)
```

**Comment by TheophileDiot** [comment]
Hi @gspiliotis, thank you for this issue, I'll check for the typo and it'll be fixed in the next hotfix or release. 😁

**Comment by fl0ppy-d1sk** [comment]
Hello @gspiliotis,

Fixed in the last [1.4.7 release](https://github.com/bunkerity/bunkerweb/releases/tag/v1.4.7). Thanks for your report.
